### PR TITLE
update docs for get Item by id

### PR DESCRIPTION
**Pull request by jaydeep0621**
Get Item By Id
image

### CODE REVIEW

File: api-reference/endpoint/get_item.mdx [hunk 1]
```diff
@@ -1,85 +1,59 @@
...
+</ResponseField>
+
 </ResponseField>
 
 <RequestExample>
 
 ```bash Example Request
-curl --location --request GET 'https://api.mintlify.com/api/user' \
+curl --location --request GET \
+'https://api.mintlify.com/api/v1/items/:itemId' \
```
Comment (reneerrusso) [inline api-reference/endpoint/get_item.mdx]: we should be calling our api url @jaydeep0621

Comment (jaydeep0621) [reply api-reference/endpoint/get_item.mdx]: Ok let me update it

Comment (jaydeep0621) [reply api-reference/endpoint/get_item.mdx]: Updated API URL

Comment (reneerrusso) [reply api-reference/endpoint/get_item.mdx]: this api works when you hit it?

**Pull request merged**